packages/inter-protocol/src/proposals/startPSM.js

```diff
@@ -217,15 +227,21 @@ export const makeAnchorAsset = async (
 };
 harden(makeAnchorAsset);
 
-/** @param {BootstrapSpace & { devices: { vatAdmin: any }, vatPowers: { D: DProxy }, }} powers */
+/** @typedef {import('./econ-behaviors.js').EconomyBootstrapSpace} EconomyBootstrapSpace */
+
+/** @param {BootstrapSpace & EconomyBootstrapSpace & { devices: { vatAdmin: any }, vatPowers: { D: DProxy }, }} powers */
+// /** @param {BootstrapSpace & { devices: { vatAdmin: any }, vatPowers: { D: DProxy }, }} powers */
 export const installGovAndPSMContracts = async ({
   vatPowers: { D },
   devices: { vatAdmin },
   consume: { zoe },
+  produce: { psmFacets },
   installation: {
     produce: { contractGovernor, committee, binaryVoteCounter, psm },
   },
 }) => {
+  psmFacets.resolve(makeScalarMapStore());
```

Comment:
Perhaps leave a comment for our future selves about what this is doing here?

Reply:
done